tools/src/commands/CherryPick.ts

```diff
@@ -0,0 +1,196 @@
+import { Command } from '@expo/commander';
```

Comment:
Could you rename this file to `CherryPickCommand.ts`? Even though this filename is not commonly used in the repo, it would be a nice standard that makes it easier to find out the right file in the file search.